test/policy-store.test.ts

```diff
@@ -15,19 +17,71 @@ import {
   READ_ACTIONS,
   WRITE_ACTIONS,
 } from '../src/private/permissions';
-import { Statement } from '../src/statement';
 
-const cedarJsonSchemaExample = {
-  PhotoApp: {
-    entityTypes: {
-      User: {},
-      Photo: {},
-    },
-    actions: {
-      viewPhoto: {
-        appliesTo: {
-          principalTypes: ['User'],
-          resourceTypes: ['Photo'],
+const exampleSchema: cedar.Schema = {
```

Comment:
still keeping it like this in the test file because the autocompletion help is very very nice to have, and maybe it can serve as an example for people using typescript